spec/requests/contributors_spec.rb

```diff
@@ -10,23 +10,55 @@
   let(:user) { create(:user, organizations: [organization]) }
 
   describe 'GET /index' do
+    let!(:other_organizations_contributor) { create(:contributor, first_name: 'WhatAName') }
+    let!(:other_inactive_contributor) { create(:contributor, :inactive) }
+    let!(:other_unsubscribed_contributor) { create(:contributor, :unsubscribed) }
+
+    before { get organization_contributors_url(organization, as: user) }
+
     it 'should be successful' do
-      get organization_contributors_url(organization, as: user)
       expect(response).to be_successful
     end
+
+    it 'contains only the contributors from the organization' do
+      expect(page).to have_content contributor.first_name
+
+      expect(page).to have_content('Aktiv 1')
+      expect(page).to have_content('Inaktiv 0')
+      expect(page).to have_content('Abbestellt 0')
+
+      expect(page).not_to have_content other_organizations_contributor.first_name
```

Comment:
Why did you add the first_name to the let, but then not use this hard-coded value here?
I know @roschaefer favors using the hard-coded values. For me, I would expect either to give a hard-coded value that I then use to assert against, or don't give the value and use the `contributor.first_name` approach.

Reply:
relying on faker for the names makes the test flaky (i've encountered an error once).. and for me it is most descriptive this way.. (it should not contain the first name of the other contributor) whereas having only the name hardcoded would need the reader to reverse lookup the value to understand what it belongs to.